MiscVBAFunctions/Modules/MiscArray.bas

```diff
@@ -36,6 +40,13 @@ Public Function ArrayToRange( _
     EndRow = StartRow + UBound(Data) - LBound(Data)
     EndColumn = StartColumn + UBound(Data, 2) - LBound(Data, 2)
     
+    If IncludesHeader Then
```

Comment:
Why not do this in `ArrayToNewTable` and remove this additional argument from `ArrayToRange`?

Reply:
Two reasons:

- Less code duplication if we use ArrayToRange for something else where the first row is also considered to be a header, e.g. for creating a Named Range table.
- The ranges are already calculated in ArrayToRange, so this keeps similar logic together, which is easier to maintain.